Makefile

```diff
@@ -1,16 +1,22 @@
+PLATFORMS := darwin/386 darwin/amd64 linux/386 linux/amd64 linux/arm \
```

Comment:
I don't think the current code base support windows platform, there is a separate PR open to add support for windows platform https://github.com/git-hooks/git-hooks/pull/23